test/TestServerProjects/media_types/Generated/Models/ContentType1.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace media_types.Models
+{
+    /// <summary> Content type for upload. </summary>
+    public enum ContentType1
```

Comment:
Is it ok that we have name like `ContentType1`?

Reply:
My concern is that it may be renamed to `ContentType2` when another schema is added to the yaml.